app/controllers/field_slips_controller.rb

```diff
@@ -219,13 +226,9 @@ def update_notes_fields(notes)
     end
   end
 
-  def collector
-    user_str(params[:field_slip][:collector])
-  end
+  def collector = user_str(params[:field_slip][:collector])
 
-  def field_slip_id_by
-    user_str(params[:field_slip][:field_slip_id_by])
-  end
+  def field_slip_id_by = user_str(params[:field_slip][:field_slip_id_by])
```

Comment:
This method `user_str` that parses `Jason Hollinger <jason>` to `Jason Hollinger` is repeated at several points in the code, including in pattern search. Can we please make this either
- A method of `User`
- A method of `StringExtensions`
and use that method everywhere in MO?

Reply:
I agree ti would be good to keep things DRY, but I think it should be in a different PR.
I'm trying to keep this PR tiny so that there's a shot at getting it deployed in the next 24 hours.